WordPress/src/main/java/org/wordpress/android/ui/mysite/SiteStoriesHandler.kt

```diff
@@ -0,0 +1,105 @@
+package org.wordpress.android.ui.mysite
+
+import android.content.Intent
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.wordpress.stories.compose.frame.FrameSaveNotifier
+import com.wordpress.stories.compose.frame.StorySaveEvents
+import com.wordpress.stories.compose.frame.StorySaveEvents.StorySaveProcessStart
+import com.wordpress.stories.compose.frame.StorySaveEvents.StorySaveResult
+import com.wordpress.stories.compose.story.StoryRepository
+import org.greenrobot.eventbus.EventBus
+import org.greenrobot.eventbus.Subscribe
+import org.greenrobot.eventbus.ThreadMode
+import org.wordpress.android.R
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.STORY_SAVE_ERROR_SNACKBAR_MANAGE_TAPPED
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.PagePostCreationSourcesDetail
+import org.wordpress.android.ui.mysite.SiteNavigationAction.OpenStories
+import org.wordpress.android.ui.pages.SnackbarMessageHolder
+import org.wordpress.android.ui.stories.StoriesMediaPickerResultHandler
+import org.wordpress.android.ui.stories.StoriesTrackerHelper
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.ui.utils.UiString.UiStringText
+import org.wordpress.android.util.EventBusWrapper
+import org.wordpress.android.util.merge
+import org.wordpress.android.viewmodel.ContextProvider
+import org.wordpress.android.viewmodel.Event
+import org.wordpress.android.viewmodel.ResourceProvider
+import javax.inject.Inject
+
+class SiteStoriesHandler
+@Inject constructor(
+    private val eventBusWrapper: EventBusWrapper,
+    private val resourceProvider: ResourceProvider,
+    private val storiesTrackerHelper: StoriesTrackerHelper,
+    private val contextProvider: ContextProvider,
+    private val selectedSiteRepository: SelectedSiteRepository,
+    private val storiesMediaPickerResultHandler: StoriesMediaPickerResultHandler
+) {
+    private val _onSnackbar = MutableLiveData<Event<SnackbarMessageHolder>>()
+    val onSnackbar = _onSnackbar as LiveData<Event<SnackbarMessageHolder>>
+    private val _onNavigation = MutableLiveData<Event<SiteNavigationAction>>()
+    val onNavigation = merge(_onNavigation, storiesMediaPickerResultHandler.onNavigation)
+
+    init {
+        eventBusWrapper.register(this)
+    }
+
+    fun clear() {
+        eventBusWrapper.unregister(this)
+    }
+
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    fun onEventMainThread(event: StorySaveResult) {
+        EventBus.getDefault().removeStickyEvent(event)
```

Comment:
Shouldn't we use `eventBusWrapper` here instead?
```suggestion
        eventBusWrapper.removeStickyEvent(event)
```